content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -97,8 +97,8 @@ For example, to download version {{< param "fullversion" >}} on Linux, type:
    ```
    Or use this for detailed view of version:
 
-   ```cmd
-   kubectl version --client --output=yaml    
```

Comment:
In documentation, we prefer long option names (eg `--option`) over short ones (`-o`). The long names are easier to memorize.

What we could do instead: link to a page that explains how different output formats work - eg https://kubernetes.io/docs/reference/kubectl/overview/#output-options

Reply:
Changed as per your suggestion